src/DeleteOldSoftDeletes.php

```diff
@@ -59,10 +59,10 @@ private function deleteOldSoftDeletes()
 
     private function deleteOldSoftDeletesForModel($modelName, $modelConfig, $daysBeforeDeletion)
     {
-        $daysBeforeDeletion = empty($modelConfig['days']) ? $daysBeforeDeletion : $modelConfig['days'];
+        $daysBeforeDeletion = $modelConfig['days'] ?? $daysBeforeDeletion;
```

Comment:
If we're going to do this (which is fine by me) I think we should add a PHP version constraint to `composer.json`

Reply:
Will do 👍